test-tool/test_prout_reserve_simple.c

```diff
@@ -29,22 +29,34 @@
 /*
  * list of persistent reservation types to test, in order
  */
-static enum scsi_persistent_out_type pr_types_to_test[] = {
-        SCSI_PERSISTENT_RESERVE_TYPE_WRITE_EXCLUSIVE,
-        SCSI_PERSISTENT_RESERVE_TYPE_EXCLUSIVE_ACCESS,
-        SCSI_PERSISTENT_RESERVE_TYPE_WRITE_EXCLUSIVE_REGISTRANTS_ONLY,
-        SCSI_PERSISTENT_RESERVE_TYPE_EXCLUSIVE_ACCESS_REGISTRANTS_ONLY,
-        SCSI_PERSISTENT_RESERVE_TYPE_WRITE_EXCLUSIVE_ALL_REGISTRANTS,
-        SCSI_PERSISTENT_RESERVE_TYPE_EXCLUSIVE_ACCESS_ALL_REGISTRANTS,
-        0
+static struct test_prin_report_caps_types {
+	enum scsi_persistent_reservation_type_mask mask;
+	enum scsi_persistent_out_type op;
+} report_caps_types_array[] = {
```

Comment:
Is this array identical to the array in test_prin_report_caps.c? If so, can this duplication be avoided?

Reply:
I more meant putting the array in iscsi-support.c then doing a helper and exporting the helper in the header. It would be like sdev_states and scsi_device_state_name the kernel.